net-mgmt/configsync/src/opnsense/mvc/app/views/VerbNetworks/ConfigSync/settings.volt

```diff
@@ -0,0 +1,206 @@
+                </p>
+<pre>{
+    "Version": "2012-10-17",
+    "Statement": [
+        {
+            "Effect": "Allow",
+            "Action": [ "s3:ListBucket" ],
+            "Resource": [ "arn:aws:s3:::BUCKET_NAME" ]
+        },
+        {
+            "Effect": "Allow",
+            "Action": [ "s3:ListBucket", "s3:PutObject" ],
+            "Resource": [ "arn:aws:s3:::BUCKET_NAME/PATH_NAME/*" ]
+        }
+    ]
+}</pre>
+                <p><strong>Note:</strong> Be sure to replace <code>BUCKET_NAME</code> and <code>PATH_NAME</code> with your own values above.</p>
+
+                <hr />
+                
+                <h1>Author</h1>
+                <p>Configuration Sync is a Verb Networks plugin for OPNsense - we make other tools for OPNsense too!</p>
+
+                <h1>License</h1>
+                <p>BSD-2-Clause - see LICENSE file for full details.</p>
+
+                <h1>Copyright</h1>
+                <p>Copyright 2018 - All rights reserved - <a href="https://verbnetworks.com/">Verb Networks Pty Ltd</a></p>
+                
+            </div>
+
+        </div>
+    </div>
+</div>
+
+<style>
+    #configsync\.settings\.StorageFullURI, #configsync\.settings\.SystemHostid {
+        line-height: 34px;
+        display: inline-block;
+        vertical-align: middle;
+        
+        font-size: 80%;
+        font-family: Menlo, Monaco, Consolas, "Courier New", monospace;
+    }
+</style>
+
+<script>
+    
+    /**
+     * updateFullURI
+     */
+    function updateFullURI() {
+        if($('#configsync\\.settings\\.Provider').val() === 'awss3') {
+            var link_url = 'https://s3.console.aws.amazon.com/s3/buckets/';
+            link_url += $('#configsync\\.settings\\.StorageBucket').val() + '/';
+            link_url += $('#configsync\\.settings\\.StoragePath').val() + '/';
+
+            var link_name = 's3://';
+            link_name += $('#configsync\\.settings\\.StorageBucket').val() + '/';
+            link_name += $('#configsync\\.settings\\.StoragePath').val();
+        }
+        $('#configsync\\.settings\\.StorageProviderLink').html('<a target="_blank" href="' + link_url +'">' + link_name + '</a>');
+    }
+    
+    /**
+     * $(document).ready
+     */
+    $(document).ready(function() {
+        
+        updateServiceControlUI('configsync');
+        
+        $("#configsync\\.settings\\.StorageBucket").change(function(){
+            updateFullURI();
+        });
+        
+        $("#configsync\\.settings\\.StoragePath").change(function(){
+            updateFullURI();
+        });
+        
+        mapDataToFormUI({frm_Settings:"/api/configsync/settings/get"}).done(function(data){
+            formatTokenizersUI();
+            $('.selectpicker').selectpicker('refresh');
+            updateFullURI();
+        });
+
+        $("#testAction").click(function(){
+            $("#responseMsg").removeClass("hidden").removeClass("alert-danger").addClass('alert-info').html("Running tests...");
```

Comment:
gettext (lang._)

Reply:
Yes and thanks, too easy to forget - Done